app/util.py

```diff
@@ -63,3 +80,51 @@ def to_json(model):
 
 def get_now():
     return int(datetime.datetime.utcnow().timestamp())
+
+PERSISTENT_FIELDS = [
+    'socials_instagram',
+    'socials_snapchat',
+    'privacy_hide_image',
+    'privacy_hide_email',
+    'privacy_hide_room',
+    'privacy_hide_phone',
+    'privacy_hide_address',
+    'privacy_hide_major',
+    'privacy_hide_birthday'
+]
+
+def scrub_hidden_data(person):
+    if hasattr(person, "privacy_hide_image") and getattr(person, "privacy_hide_image"):
+        setattr(person, "image", None)
+    if hasattr(person, "privacy_hide_email") and getattr(person, "privacy_hide_email"):
+        setattr(person, "email", None)
+    if hasattr(person, "privacy_hide_room") and getattr(person, "privacy_hide_room"):
+        setattr(person, "entryway", None)
+        setattr(person, "floor", None)
+        setattr(person, "suite", None)
+        setattr(person, "room", None)
+    if hasattr(person, "privacy_hide_phone") and getattr(person, "privacy_hide_phone"):
+        setattr(person, "phone", None)
+    if hasattr(person, "privacy_hide_address") and getattr(person, "privacy_hide_address"):
+        setattr(person, "address", None)
+    if hasattr(person, "privacy_hide_major") and getattr(person, "privacy_hide_major"):
+        setattr(person, "major", None)
+    if hasattr(person, "privacy_hide_birthday") and getattr(person, "privacy_hide_birthday"):
+        setattr(person, "birthday", None)
+        setattr(person, "birth_month", None)
+        setattr(person, "birth_day", None)
+    return person
+
+INSTAGRAM_ALLOWED_CHARSET = set(string.ascii_letters + string.digits + '._')
```

Comment:
I think this would fail the style test - can you run `./style_test.sh` and fix any issues that pop up?